css/component_styles/grid.css

```diff
@@ -1,3 +1,52 @@
 @import url('./components.css');
 
-/* -------- grid css -------- */
\ No newline at end of file
+/* -------- grid css -------- */
+
+.grid-container {
+    display: grid;
+    background-color: var(--bg-color);
+    width: 100%;
+}
+
+.grid-item {
+    border: solid 2px var(--color-dnd);
```

Comment:
I hope you have used the same format for border at other places also, for consistency

Reply:
Thanks